hudi-common/src/main/java/org/apache/hudi/common/util/RocksDBSchemaHelper.java

```diff
@@ -78,12 +78,12 @@ public String getPrefixForSliceViewByPartitionFile(String partitionPath, String
     return String.format("type=slice,part=%s,id=%s,instant=", partitionPath, fileId);
   }
 
-  public String getPrefixForDataFileViewByPartitionFile(String partitionPath, String fileId) {
-    return String.format("type=df,part=%s,id=%s,instant=", partitionPath, fileId);
+  public String getPrefixForBaseFileViewByPartitionFile(String partitionPath, String fileId) {
+    return String.format("type=bf,part=%s,id=%s,instant=", partitionPath, fileId);
```

Comment:
Hmmm. this should be since everytime we deploy, we rebuild the rocksdb cache? @bvaradar ?

Reply:
Yes, this is currently transient. It is fine.